draft-wang-ppm-differential-privacy.md

```diff
@@ -456,9 +473,74 @@ class DpMechanism:
 
 > TODO: Specify a Gaussian sampler
 
-## Randomized Response {#rr}
+## Binary Randomized Response {#rr}
+
+The binary randomized response mechanism specified in Appendix C.1 of
+{{MJTB+22}} is `EPSILON_0`-DP at the Client level, in the deletion-DP model
+(Definition II.4 of {{EFMR+20}}).
+
+Binary randomized response's input is a single bit `x`. The bit is flipped to
+`1 - x` with probability `1 / (exp(EPSILON_0) + 1)`.
+
+We generalize this noise by applying it independently at all coordinates of a
+Client's bit vector. Under OAMC trust model, it is proven in Appendix C.1.3
+of {{MJTB+22}} that we get good `(EPSILON, DELTA)`-DP in the replacement-DP
```

Comment:
"replacement-DP" is defined below (in the appendix) but not above.  If you want to use the term, make sure it's defined before reader gets here. (I don't think it's strictly necessary, since we defined DP in terms of replacement already.)

Reply:
Good point. Move that part of the text to overview now, where we've been defining `EPSILON`-DP and `(EPSILON, DELTA)`-DP in terms of replacement-DP. I didn't go in depth on deletion-DP, which I only elaborated in the section for binary randomized response.